package.json

```diff
@@ -4,64 +4,63 @@
   "private": true,
   "type": "module",
   "dependencies": {
-    "@azure/msal-browser": "^3.10.0",
-    "@azure/msal-react": "^2.0.12",
-    "@emotion/react": "^11.11.3",
-    "@emotion/styled": "^11.10.8",
-    "@mui/icons-material": "^5.15.10",
-    "@mui/material": "^5.15.10",
-    "@mui/x-date-pickers": "^6.19.4",
-    "add-to-calendar-button": "^2.5.10",
-    "caniuse-lite": "^1.0.30001589",
-    "chart.js": "^2.9.4",
-    "cropperjs": "^1.6.1",
-    "date-fns": "^3.3.1",
+    "@azure/msal-browser": "^3.16.0",
+    "@azure/msal-react": "^2.0.18",
```

Comment:
Thanks for thinking about this @russtuck. To answer your first question, our dependencies have specified patch versions because that is how `npm` installs them.

For example, if I run `npm install react` in a brand new project, `npm` puts the following in my `package.json`:
```json
{
  "dependencies": {
    "react": "^18.3.1"
  }
}
```

So you can see that `npm`s default behavior is to install the package specified to the latest patch, but with a Caret Range specifying that the package can be automatically updated to any version that isn't expected to be a breaking change. This is (almost) equivalent to `react: 18`. The one exception is that Caret Ranges handle pre-1.0 packages more carefully.

Reply:
I just updated by using a package updater. I can probably just make it `3.x.x` or whatever. Solid find